lib/src/html_renderer.dart

```diff
@@ -19,13 +19,19 @@ String markdownToHtml(
   Resolver? linkResolver,
   Resolver? imageLinkResolver,
   bool inlineOnly = false,
+  bool? encodeHtml,
```

Comment:
Can we use default values here, `bool encodeHtml = true`, to avoid the nullable bool?

Reply:
Agreed. Fixed